hazelcast/src/test/java/com/hazelcast/map/impl/eviction/ExpirationManagerTest.java

```diff
@@ -1,431 +0,0 @@
-import com.hazelcast.map.impl.MapService;
-import com.hazelcast.map.impl.PartitionContainer;
-import com.hazelcast.map.listener.EntryExpiredListener;
-import com.hazelcast.test.AssertTask;
-import com.hazelcast.test.HazelcastSerialClassRunner;
-import com.hazelcast.test.HazelcastTestSupport;
-import com.hazelcast.test.TestHazelcastInstanceFactory;
-import com.hazelcast.test.annotation.ParallelTest;
-import com.hazelcast.test.annotation.QuickTest;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.rules.ExpectedException;
-import org.junit.runner.RunWith;
-
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import static com.hazelcast.cluster.ClusterState.ACTIVE;
-import static com.hazelcast.cluster.ClusterState.PASSIVE;
-import static com.hazelcast.core.LifecycleEvent.LifecycleState.MERGED;
-import static com.hazelcast.core.LifecycleEvent.LifecycleState.MERGING;
-import static com.hazelcast.core.LifecycleEvent.LifecycleState.SHUTTING_DOWN;
-import static com.hazelcast.map.impl.MapService.SERVICE_NAME;
-import static com.hazelcast.map.impl.eviction.ExpirationManager.PROP_CLEANUP_OPERATION_COUNT;
-import static com.hazelcast.map.impl.eviction.ExpirationManager.PROP_CLEANUP_PERCENTAGE;
-import static com.hazelcast.map.impl.eviction.ExpirationManager.PROP_PRIMARY_DRIVES_BACKUP;
-import static com.hazelcast.map.impl.eviction.ExpirationManager.PROP_TASK_PERIOD_SECONDS;
-import static com.hazelcast.map.impl.eviction.ExpirationManagerStressTest.getExpirationManager;
-import static java.lang.Integer.parseInt;
-import static java.lang.String.format;
-import static java.lang.String.valueOf;
-import static java.lang.System.clearProperty;
-import static java.lang.System.getProperty;
-import static java.lang.System.setProperty;
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
-@RunWith(HazelcastSerialClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
-public class ExpirationManagerTest extends HazelcastTestSupport {
-
-    @Rule
-    public ExpectedException thrown = ExpectedException.none();
-
-    @Test
-    public void testTaskPeriodSeconds_set_viaSystemProperty() {
-        String previous = getProperty(PROP_TASK_PERIOD_SECONDS);
-        try {
-            int expectedPeriodSeconds = 12;
-            setProperty(PROP_TASK_PERIOD_SECONDS, valueOf(expectedPeriodSeconds));
-
-            int actualTaskPeriodSeconds = newExpirationManager(createHazelcastInstance()).getTaskPeriodSeconds();
-
-            assertEquals(expectedPeriodSeconds, actualTaskPeriodSeconds);
-        } finally {
-            restoreProperty(PROP_TASK_PERIOD_SECONDS, previous);
-        }
-    }
-
-    @Test
-    public void testPrimaryDrivesEvictions_set_viaSystemProperty() {
```

Comment:
It seems that this test has not been migrated to `AbstractExpirationManagerTest` or its descendants. Is it intended?

Reply:
good catch. It slipped away when refactoring. I'll add it back.